packages/performance-tests/test/transformers/RawForkCreateFedGuids.ts

```diff
@@ -1,15 +1,15 @@
-import { initializeBranchProvenance } from "@itwin/imodel-transformer";
+import * as fs from "fs";
+import { IModelDb, StandaloneDb } from "@itwin/core-backend";
 import { TestTransformerModule, TransformRunner } from "../TestTransformerModule";
 import { initOutputFile } from "../TestUtils";
-import * as fs from "fs";
+import { initializeBranchProvenance } from "@itwin/imodel-transformer";
 import { setToStandalone } from "../iModelUtils";
 import path from "path";
-import { StandaloneDb } from "@itwin/core-backend";
 
 const outputDir = path.join(__dirname, ".output");
 
 const rawForkCreateFedGuidsTestModule: TestTransformerModule = {
-  async createForkInitTransform(sourceDb, targetDb): Promise<TransformRunner> {
+  async createForkInitTransform(sourceDb: IModelDb, targetDb: IModelDb): Promise<TransformRunner> {
```

Comment:
why are these changes necessary?

Reply:
`NativeTransformer` has those specified, so I thought it would be better to make it clear what type are source and target dbs.